modules/service/src/main/scala/lucuma/itc/service/ItcMappings.scala

```diff
@@ -58,16 +58,8 @@ object ItcMapping extends ItcCacheOrRemote with Version {
     Result(ItcVersions(version(environment).value, BuildInfo.ocslibHash.some)).pure[F]
 
   private val buildError: Throwable => Error =
-    case SourceTooBright(hw)       => Error.SourceTooBright(hw)
-    case UpstreamException(m)      => Error.General(m.mkString("\n"))
```

Comment:
We maybe using this format to get better messages on the frontend

Reply:
> We maybe using this format to get better messages on the frontend

The newlines are already being added in the constructor, the goal here is to add a prefix to the messages so that we know where they originate.